FSJ_django20_project/FSJ/forms_fileupload.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+class FileUploadForm(forms.Form):
+    student_file = forms.FileField(required = False, help_text = ("Upload a CSV with fields CCID, ID, First Name, Last Name, Email (Univ), Prog & Year"))
```

Comment:
There is not translation for the English text here. Normally, we import gettext or gettext_lazy and alias it as _ so the syntax would be _("text") not ("text")